### PR TITLE
Companion PR for Substrate #5375

### PR DESCRIPTION
**Pull request by octol**
https://github.com/paritytech/substrate/pull/5375

### CODE REVIEW

File: rpc/Cargo.toml [hunk 1]
```diff
@@ -12,6 +12,9 @@ sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "mas
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master"  }
 sp-api = { git = "https://github.com/paritytech/substrate", branch = "master"  }
 sc-rpc = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-finality-grandpa = { git = "https://github.com/paritytech/substrate", branch = "master" }
+# FIXME: must be updated before merging
```
Comment (andresilva) [inline rpc/Cargo.toml:16]: This must be updated before merging.

File: rpc/Cargo.toml [hunk 2]
```diff
@@ -12,6 +12,9 @@ sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "mas
...
+sc-finality-grandpa-rpc = { git = "https://github.com/octol/substrate", branch = "jon/issue-4921-expose-grandpa-round-state-try-shared-arc" }
```
Comment (gnunicorn) [inline rpc/Cargo.toml:17]: Update for merge
```suggestion
sc-finality-grandpa-rpc = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Comment (andresilva) [reply rpc/Cargo.toml]: I made this temporary change purposefully. The current approach of overriding with `.cargo/config` doesn't work when a crate doesn't exist (i.e. sc-finality-grandpa-rpc currently does not exist on master). Without this change the substrate companion build task will fail to build.

Comment (gnunicorn) [reply rpc/Cargo.toml]: yeah, just learned that the hard way. but we had CI pass, so if the review passes, we can merge non-the-less.

**Pull request merged**